gateway/src/main/java/io/zeebe/gateway/EndpointManager.java

```diff
@@ -251,6 +249,11 @@ public void topology(
           .setPartitionsCount(topology.getPartitionsCount())
           .setReplicationFactor(topology.getReplicationFactor());
 
+      final String gatewayVersion = getClass().getPackage().getImplementationVersion();
```

Comment:
just a nit pick, feel free to ignore, afaik if the version is blank is that that's what we set when we built the JAR, so not sure if it's all that important to check it here.

Reply:
If we set the version to null, it will throw an NPE. I agree that if it's an empty string it's not serious since the clients already check and deal with that. But I thought it'd be better to include this since this way the clients don't have to deal with random whitespace, since the version will be either something or an empty string

Reply:
Fair enough